proxy/doc.go

```diff
@@ -70,6 +70,10 @@ Routing will happen again with an augmented request.
 This can happen several times in a row until maxLoopbacks
 limit is reached.
 
+3.d forward:
+
+In case of a `forward` route, requests are forwarded to the original destination.
```

Comment:
original destination got from `Host: destination`, right?
I think this should be in the docs

Reply:
`forward` is now `dynamic`